### ISSUE TITLE
Styling tatoolChoiceReaction

### ISSUE THREAD

**Issue by andreadecesarei** [tatool/tatool-web#132]
Hi, 

I am trying to build an experiment with tatool, everything works fine. Congrats for this excellent piece of software! However at a point of my design, I have to display a long text on every trial and collect the response. To this end, I use tatoolChoiceReaction but I need to use a smaller font size compared to the "standard" one (550% if I understand correctly the css file). How can I do? 

Many thanks in advance, 
Andrea

PS moreover, if I create the experiment using the web interface, the executable folder is empty, which creates me some confusion if e.g. I want to use the information which is in the tutorial for styling the experiment

**Comment by alocher** [comment]
Hi Andrea, it's great to hear that Tatool is useful for you!

As for the styling you have two options:

- Provide a stimulusValueStyle column in your stimuli file and provide a css styling (e.g. color:red;font-size:15px". See the documentation about the Tatool Stimuls Template Service for the possible parameters available (https://www.tatool-web.com/#/doc/ref-directive-tatoolStimulus.html)
- Change your stimuli to be images instead, which allows you to layout everything the way you want to.

I am not entirely sure I understand your second question. Do you mind explaining this in a bit more detail?

**Comment by andreadecesarei** [comment]
Hi, many thanks for the prompt response! 

Yes, I tried changing the stimulusValueStyle  column. However, I see no change in how the text appears - am I missing something? 
Also, I noticed that it is possible to customize text by adding curly brackets for session-level variables (eg., {{session}}). Is it also possible  to reference columns in the stimuli file by the same or similar mechanisms?

the csv file looks like this: 
stimulusValue;stimulusValueType;stimulusValueStyle ;correctResponse;keyCode;response
Benvenuto! Nel seguente esperimento ti saranno mostrate immagini. Dovrai guardare le immagini e dopo ognuna di esse valutare quanto intenso e quanto piacevole  stato lo stato emozionale indotto dall'immagine. ;text;"color:red;font-size:15px";Space;Space;ok

thanks!
Best, Andrea

**Comment by alocher** [comment]
I have just tried myself and you are right that the styling doesn't work right now. It looks like you uncovered a bug there :). I will have a look at this and let you know once fixed. In the meantime you could use an image instead if that works for now.

The curly brackets parameters are currently only available in instruction html files. So that would either be you running your own instance of tatool-web which allows you to add whatever you want or we would have to host the html files for you as we currently can't allow external html files.

**Comment by andreadecesarei** [comment]
Hi, thanks for the very quick responses! Good that I helped spotting a bug. Once again, I really appreciate the effort put into this software. I will try to build some obvious and less obvious paradigms, but it already looks extremely promising! 
Best, 
Andrea

**Comment by alocher** [comment]
This should now be fixed. Let me know if you still encounter any issues. I noticed in the example stimuli file you provided you had a space after "stimulusValueStyle". Make sure to remove that as the header name needs to match exactly.

**Comment by andreadecesarei** [comment]
Hi,

I just tried the new version, it works perfectly!

Thanks you so much for your support.

Best,
Andrea

Andrea De Cesarei, PhD
Associate Professor
Alma Mater Studiorum-University of Bologna
Department of Psychology
Viale Berti Pichat 5
40127 Bologna
ITALY
e-mail: andrea.decesarei@unibo.it<mailto:andrea.decesarei@unibo.it>
Website: http://goo.gl/KoY3O

Da: Andre <notifications@github.com>
Inviato: venerdì 27 marzo 2020 17:41
A: tatool/tatool-web <tatool-web@noreply.github.com>
Cc: Andrea De Cesarei <andrea.decesarei@unibo.it>; Author <author@noreply.github.com>
Oggetto: Re: [tatool/tatool-web] Styling tatoolChoiceReaction (#132)


This should now be fixed. Let me know if you still encounter any issues. I noticed in the example stimuli file you provided you had a space after "stimulusValueStyle". Make sure to remove that as the header name needs to match exactly.
